cloudinit/sources/DataSourceExoscale.py

```diff
@@ -79,6 +79,7 @@ def crawl_metadata(self):
         metadata_ready = self.wait_for_metadata_service()
 
         if not metadata_ready:
+            LOG.error("Unable to get response from metadata service")
```

Comment:
Do we really need datasource-specific error messages?

We should have unified warnings when `get_data()` fails, I would expect.

Reply:
We already have datasource specific warnings (or non-warning logs). Look at all the other invocations of `wait_for_url()`.

Reply:
Yeah, I get that. It just seems redundant with errors reported further up the stack but I guess it doesn't hurt.